src/content/docs/apis/nerdgraph/examples/nerdgraph-api-secret-management-service.mdx

```diff
@@ -0,0 +1,255 @@
+---
+title: "NerdGraph tutorial: Secret Management Service"
+metaDescription: "Learn how to use the Secret Management Service to securely store and manage your secrets."
+freshnessValidatedDate: never
+---
+
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+The Secret Management Service is a secure and scalable way to store and manage your secrets. It provides a centralized location for storing and managing secrets, such as API keys to reduce fragmentation and complexity across different services in New Relic. 
+You can store any string value as a secret, and call them programatically during runtime.
+
+Note that there's no data boundary for storing secrets; all secrets reside in US in a global DB enclave.
+
+Use the [NerdGraph API](https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) to interact with the Secret Management Service. This tutorial will guide you through the process of creating, updating, and deleting secrets using the Secret Management Service.
+
+## Key features
+
+* **Centralized Repository**: Store all your confidential keys and credentials in one secure location.
+* **Secure Management**: Enhanced security features to protect your secrets.
+* **Scalability**: Designed to handle the growing needs of enterprises.
+
+## Requirements[#review-requirements]
+ 
+* You need a [New Relic account](https://newrelic.com/signup), and with that account, you can access your API user key that you need to include with queries and mutations.
+
+* User type and assigned roles can affect your NerdGraph permissions. For more details, see [Factors affecting access](https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data/#user-permissions).
+
+## Mutation operations
+
+To enable users to interact with the Secrets Management Service, following are the exposed four NerdGraph mutations:
+
+
+### Create a secret[#create-secret]
+
+The `secretsManagementCreateSecret` mutation allows you to add new secrets to the Secrets Management Service.
+
+#### Input Parameters
+
+* `accountId` (String, required): The account ID associated with the secret.
+* `alias` (String, optional): An alias for the secret.
+* `entityGuid` (String, optional): The entity GUID associated with the secret.
+* `key` (String, required): The key for the secret.
+* `namespace` (String, required): The namespace to which the secret belongs.
```

Comment:
I think we need to add a sentence explaining what does it mean when we say a secret belongs to a namespace. Another reason why I think tables are better way to handle this content. 

Namespace is an abstraction that enables devs to better organize their 'secrets', in this case. We dont need to explain what namespaces are, I agree, but we should include an example as to how this is to be used, all the more so because this is a required attr.